soccer/src/soccer/strategy/agent/position/runner.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <cmath>
+
+#include <spdlog/spdlog.h>
+
+#include <rj_common/time.hpp>
+#include <rj_geometry/geometry_conversions.hpp>
+#include <rj_geometry/point.hpp>
+
+#include "position.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "rclcpp_action/rclcpp_action.hpp"
+#include "rj_msgs/action/robot_move.hpp"
+
+namespace strategy {
+
+class Runner : public Position {
```

Comment:
Note the clang-tidy warning visible in GH here. Unfortunately for virtual classes you have to define some constructors and operators manually. See the "rule of five" under this link https://en.cppreference.com/w/cpp/language/rule_of_three. Note that you can explicitly set each of these to `=default;`